src/main/java/world/bentobox/aoneblock/oneblocks/OneBlocksManager.java

```diff
@@ -3,28 +3,22 @@
 import java.io.File;
 import java.io.FilenameFilter;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
+import java.lang.reflect.Field;
```

Comment:
Reflection... let's see where this is used...

Reply:
Game Profile Field. Its the only way I know of